README.md

```diff
@@ -62,6 +62,21 @@ BeeswaxAPI::App.configure do |config|
 end
 ```
 
+Example of using:
+
+```ruby
+  params = {
+    advertiser_id: ENV["BEESWAX_ADVERTISER_ID"].to_i,
+    creative_asset_name: file_name.html,
+    size_in_bytes: 12001,
```

Comment:
is this parameter mandatory? Why is needed?

Reply:
They are mandatory for this specific operation. I wanted to show that all body param should go to `body_params` parameters.